templates/aws/nodes/terraform/pools/main.tf

```diff
@@ -37,10 +37,17 @@ resource "aws_instance" "node" {
      device_name           = "/dev/sda1"
      volume_size           = var.aws_volume_size
      volume_type           = var.aws_volume_type
-     encrypted             = true
+     encrypted             = var.aws_volume_encrypted
+     iops                  = var.aws_volume_iops
      delete_on_termination = true
    }
 
+  user_data = <<EOF
```

Comment:
What is the purpose of this?

Reply:
This is more reliable than using local-exec in my testing. This is applied during instance configuration or before boot in aws

Reply:
I still don't understand we're already doing this in our  `provisioner "remote-exec" {` I just want to understand why we would do this more than once?

Reply:
Im not sure about `remote-exec` in this file, that's only doing `root`

Reply:
I think that's going to add the keys to like `/ubuntu/.ssh/authorized_keys` which is not a valid path. I could `tee` the echo to `/home/$(whoami)/.ssh/authorized_keys` 

Is this what you mean?